### ISSUE TITLE
Feed and story pictures are not downloaded in full resolution (1350px instead of 1800px, 1080px instead of 1440px)

### ISSUE THREAD

**Issue by BlackEye2k** [HuiiBuh/InstagramDownloader#307]
# Important: Issues without the following information will be closed

**To Reproduce**
Steps to reproduce the behaviour:
1. Open any picture post (e.g. https://www.instagram.com/p/CeMCaR9KdP1/)
2. Click on the lower right download button
3. Save picture
4. Check download picture's resolution => 1080x566

**Expected behaviour**
Using browser development tools, inspect picture, (._aagv ._aagt), download, resolution => 1440x754

Expected behaviour of the plugin would be to download the provided max resolution

**Device (please complete the following information):**
- Resolution: 2560x1440
- Browser: Firefox 101.0
- Extension Version: 4.6.2

**Additional context**
This happens for stories as well

**Comment by RogGN** [comment]
I experienced the same bug by accident. I saved a story using storysaver.net, and then saved again using the IG Downloader extension. The one from storysaver.net was double the size. First I thought they were upscaling it, but if you zoom both photos you can see the blurry pixels on the photo from IG Downloader, its indeed a lower resolution.

**Comment by Gallyi** [comment]
> I experienced the same bug by accident. I saved a story using storysaver.net, and then saved again using the IG Downloader extension. The one from storysaver.net was double the size. First I thought they were upscaling it, but if you zoom both photos you can see the blurry pixels on the photo from IG Downloader, its indeed a lower resolution.

Yup, sadly, almost a month later, we still have to resort to tools such as StorySaver to manually save a post/story in high res.

**Comment by afterdelight** [comment]
use this script https://greasyfork.org/en/scripts/406535-instagram-download-button
or instaloader
